custom/icds_reports/urls.py

```diff
@@ -117,4 +115,10 @@
     url(r'^issnip_pdf_status/', CheckPDFReportStatus.as_view(), name='issnip_pdf_status'),
     url(r'^icds_image_accessor/(?P<form_id>[\w\-:]+)/(?P<attachment_id>.*)$',
         ICDSImagesAccessorAPI.as_view(), name='icds_image_accessor'),
+    url('^', DashboardView.as_view(), name='icds_dashboard')
+]
+
+urlpatterns = [
+    url(r'^tableau/(?P<workbook>\w+)/(?P<worksheet>\w+)$', TableauView.as_view(), name='icds_tableau'),
+    url(r'^icds_dashboard/', include(dashboardurls)),
```

Comment:
@lwyszomi Can you change this to only hve icds_image_accessor on this. This is a pretty big change, so I think that it would be good to only add new URLs under icds_dashboard for now, and save moving them all over until another full round of QA